src/components/designer/NodeContextMenu.tsx

```diff
@@ -54,6 +54,11 @@ const NodeContextMenu: React.FC<Props> = ({ node: { id }, children }) => {
       <MintAssetButton type={'menu'} node={node as TaroNode} />,
       isStarted && isTaro,
     ),
+    addItemIf(
+      'sendAsset',
+      <SendAssetButton type={'menu'} node={node as TaroNode} />,
+      isStarted && isTaro,
+    ),
```

Comment:
I mentioned in the previous review: i think it would be better to put "Send Asset" at the top of the context menu and Actions sidebar since I suspect this would be the most used action.